src/plugins/blocksmith/components/divider-block-placeholder/PreviewComponent.jsx

```diff
@@ -0,0 +1,26 @@
+import DividerBlockV1Mixin from '@triniti/schemas/triniti/canvas/mixin/divider-block/DividerBlockV1Mixin';
+import { localize } from '@triniti/cms/plugins/utils/services/Localization';
+import Message from '@gdbots/pbj/Message';
+import PropTypes from 'prop-types';
+import React from 'react';
+import './styles.scss';
+
+
+const PreviewComponent = ({ node }) => (
+  <div role="presentation" className="divider__container">
+    <p className={`divider__title ${!node.has('text') && 'divider__title_without_text'}`}>{localize(DividerBlockV1Mixin.findOne().getQName().getVendor())} Divider Block</p>
```

Comment:
@cohencodes when you are conditionally applying classes `classNames` is helpful

for ex this could be `className={classNames('divider__title', { divider__title_without_text: !node.has('text') })}`

that being said, is this even necessary here? as is this is what im seeing:

![Screen Shot 2019-10-09 at 5 54 08 PM](https://user-images.githubusercontent.com/6315388/66530883-85252280-eabe-11e9-8217-94012d312a03.png)

they both have a class of `false` before saving, and then after saving one has the class properly and the other has a class of `null`. either way they look styled properly, but should figure out what that weirdness is about